### PR TITLE
adapted links for authentication, renamed some files

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/authentication/tfa-bundle/tfa-introduction.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ To check whether TFA is enabled for a certain user, go to the **Users** page and
 ![TFA status](/images/users-guide/Administration/admin-tfa-sms.png)
 
 {{< c8y-admon-related >}}
-- [Changing settings > Changing authentication settings](/users-guide/administration/#authentication) for information on how to configure TFA and other authentication settings.
+- [Basic settings](/authentication/basic-settings/) for information on how to configure basic authentication settings.
```
Comment (rssag) [inline content/authentication/tfa-bundle/tfa-introduction.md:17]: shouldnt this be the full path? even though it's in the same section

File: content/authentication/basic-settings.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ For easier user access management, the above permission(s) are/is included in th
 {{< /c8y-admon-req >}}
 
 {{< c8y-admon-related >}}
-- [Two-factor authentication](/users-guide/administration/#tfa) for details on the two-factor authentication strategies in {{< product-c8y-iot >}}.
-- [Configuring single sign-on](/users-guide/administration/#configuring-single-sign-on) for details on configuring single sign-on in {{< product-c8y-iot >}}.
+- [Two-factor authentication](/authentication/tfa/) for details on the two-factor authentication strategies in {{< product-c8y-iot >}}.
```
Comment (MWindel) [inline content/authentication/basic-settings.md:41]: Should this not be the full link?

File: content/authentication/sso-bundle/sso-introduction.md [hunk 3]
```diff
@@ -7,22 +7,22 @@ section:
...
+* All microservices are built with Microservice Java SDK version 10.4.6 but preferably higher. For custom-built microservices, refer to [Security](/microservice-sdk/concept/#security).
 * For on premises installation the domain-based tenant resolution is configured properly.
 * For {{< enterprise-tenant >}}s, the enterprise domain must be set up as redirect URI in the basic configurations. If SSO providers have a list of allowed domains, the enterprise domain should be added to that list.
 * You must assign a role to the user with at least READ permission for "Own user management", otherwise the user cannot log in.
 * Users must have cookies enabled in the browser settings, as the SSO feature is built on top of cookies technology.
 {{< /c8y-admon-req >}}
 
 {{< c8y-admon-related >}}
-- [Changing settings > Changing authentication settings](/users-guide/administration/#authentication) for general information on how to configure authentication settings.
+- [Basic settings](/authentication/basic-settings/) for information on how to configure basic authentication settings.
```
Comment (MWindel) [inline content/authentication/sso-bundle/sso-introduction.md:26]: Should this not be the full path?

File: content/authentication/basic-settings.md (hunk 2 above)
Comment (BeateRixen) [inline content/authentication/basic-settings.md:41]:
```suggestion
- [Platform administration > Authentication > Two-factor authentication](/authentication/tfa/) for details on the two-factor authentication strategies in {{< product-c8y-iot >}}.
```

File: content/authentication/basic-settings.md [hunk 4]
```diff
@@ -38,8 +38,8 @@ For easier user access management, the above permission(s) are/is included in th
...
+- [Configuring single sign-on](/authentication/sso/) for details on configuring single sign-on in {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/authentication/basic-settings.md:42]:
```suggestion
- [Platform administration > Authentication > Configuring single sign-on](/authentication/sso/) for details on configuring single sign-on in {{< product-c8y-iot >}}.
```

File: content/authentication/sso-bundle/sso-introduction.md (hunk 3 above)
Comment (BeateRixen) [inline content/authentication/sso-bundle/sso-introduction.md:26]:
```suggestion
- [Platform administration > Authentication > Basic settings](/authentication/basic-settings/) for information on how to configure basic authentication settings.
```

File: content/authentication/tfa-bundle/tfa-introduction.md (hunk 1 above)
Comment (BeateRixen) [inline content/authentication/tfa-bundle/tfa-introduction.md:17]:
```suggestion
- [Platform administration > Authentication > Basic settings](/authentication/basic-settings/) for information on how to configure basic authentication settings.
```

**Review by MWindel — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**